_layouts/meltano_plugin.html

```diff
@@ -326,21 +295,47 @@ <h3 id="next-steps">Next steps</h3>
 
 <h2 id="capabilities">Capabilities</h2>
     
-{% if variant.capabilities.size == 0 %}
+{% if plugin.capabilities.size == 0 %}
 <p>
-  Check the
-  <a href="{{variant.repo}}#readme" target="_blank">README and code in the repository</a>
-  for more information on capabilities for this {{plugin_type}}.
+  <code>{{plugin.name}}</code> does not have any capabilities defined in its metadata.
+  Please considering adding them by making a PR to the
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the capabilities for this {{plugin_type}}.
 </p>
-{% else %} <ul> {% for capability in plugin.capabilities %} 
+
+{% else %} 
+
+  {% if plugin.capabilities.size < 3 %}
+    <p>
+      <code>{{plugin.name}}</code> may not have the full set of capabilities defined in its metadata.
+      Please considering adding them by making a PR to the
+      <a href="{{yaml_file}}">YAML file</a>
+      that defines the capabilities for this {{plugin_type}}.
+    </p>
+  {% endif %}
+
+<ul> {% for capability in plugin.capabilities %} 
   <li>
     <a href="/singer/docs#{{capability}}">
       <code>{{capability}}</code>
     </a>
-  </li> {% endfor %} </ul> {% endif %}
+  </li> {% endfor %} </ul> 
+
+  {% endif %}
 
-{% if plugin.settings.size > 0 %}
 <h2 id="settings">Settings</h2>
+  
+{% if plugin.settings.size == 0 %}
+<p>
+  <code>{{plugin.name}}</code> does not have any settings defined in its metadata. 
+  Please considering adding them by making a PR to the 
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the settings for this {{plugin_type}}.
+</p>
+
+{% endif %}
+
+{% if plugin.settings.size > 0 %}
```

Comment:
This can become an `else`

Reply:
Resolved via 772fa1ba19c2ce878d480ce03c63a628e39a3fa5